docker/docker-compose.yaml

```diff
@@ -130,6 +130,9 @@ services:
       - ../prompt-service/.env
     labels:
       - traefik.enable=false
+    extra_hosts:
+      # "host-gateway" is a special string that translates to host docker0 i/f IP.
+      - "host.docker.internal:host-gateway"
```

Comment:
@hari-kuriakose is this added for development purposes?

Reply:
@ritwik-g Yes. 
For local development, if `prompt service` is running in a Docker container and `platform-service` is running directly on host, then this is required so that `prompt-service` can talk to the host.